README.md

```diff
@@ -4,9 +4,10 @@ This is a [Heroku buildpack](http://devcenter.heroku.com/articles/buildpacks) th
 allows an application to use an [stunnel](http://stunnel.org) to connect securely to
 Heroku Redis.  It is meant to be used in conjunction with other buildpacks.
 
-**This buildpack is only for use with Heroku Redis 4 and 5. For Heroku Redis 6 and newer, use its built-in TLS support instead.**
-
-**For more information, see [Securing Heroku Redis](https://devcenter.heroku.com/articles/securing-heroku-redis).**
+> [!WARNING]
+> This buildpack isn’t compatible with `heroku-24` and later. You don’t need this buildpack for Redis 6+, which supports native TLS.
```

Comment:
Yeah. I was hoping the 24 page existed, maybe your right, this is a good middle ground. I like it.

Reply:
I'll be writing it soon, however, even then it won't be published publicly until it's possible to use Heroku-24 on the platform as a non-internal user, to avoid end-user confusion.